.github/workflows/deploy.yml

```
@@ -7,6 +7,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+
+    - name: Style test
+      run: |
+        make lint
+
```

Please add mypy tests:

```suggestions
    - name: Static Test
      run: |
        make static
```